test/userPantry-test.js

```diff
@@ -1,18 +1,90 @@
 const chai = require('chai');
 const expect = chai.expect;
-// const UserPantry = require('../src/userPantry');
-
-
-// describe('UserPantry', function () {
-//   let userPantry;
-//   beforeEach(function () {
-//     userPantry = new UserPantry();
-//   });
-//   it('should be a function', function () {
-//     expect(UserPantry).to.be.a('function');
-//   });
-//   it('should be an instance of UserPantry', function () {
-//     expect(userPantry).to.be.an.instanceof(UserPantry);
-//   });
-
-// });
+const RecipeRepository = require('../src/recipeRepository');
+const User = require('../src/userClass');
+const superFakeIngredientData = require('../data/fakeIngredientData');
+const fakeRecipeData = require('../data/fakeRecipeData');
+const fakeUserData = require('../data/fakeUserData');
+const Ingredient = require('../src/ingredient');
+const UserPantry = require('../src/userPantry');
+const ingredientData = require('../data/ingredients');
+
+describe('UserPantry', function () {
+  let user;
+  let userPantry;
+  let recipeRepository;
+  let recipe1;
+  let recipe2;
+  beforeEach(function () {
+    user = new User(fakeUserData[0]);
+    recipeRepository = new RecipeRepository(fakeRecipeData);
+    recipeRepository.addRecipesToRepository();
+    recipe1 = recipeRepository.recipeList[0];
+    recipe2 = recipeRepository.recipeList[1];
+    userPantry = new UserPantry(user);
+  });
+
+  it('should be a function', function () {
+    expect(UserPantry).to.be.a('function');
+  });
+
+  it('should be an instance of UserPantry', function () {
+    expect(userPantry).to.be.an.instanceof(UserPantry);
+  });
+
+  it('should start off with an empty neededIngredients array', function () {
+    expect(userPantry.neededIngredients).to.deep.equal([]);
+  })
+
+  it('should have items in the user pantry', function () {
+    expect(userPantry.pantry.length).to.deep.equal(user.pantry.length);
```

Comment:
you can just equal for length. I would add deep.equal to check the actual content of the array.